webserver/views/user.py

```diff
@@ -6,6 +6,8 @@
 from datetime import datetime
 import webserver
 import db.user
+import ujson
```

Comment:
Why did you decide to use `ujson` package instead of `json`? We encountered some problems with accuracy when converting some values in AcousticBrainz using `ujson`. Do you know if this is relevant here?